spec/paperclip/storage/s3_spec.rb

```diff
@@ -14,6 +14,21 @@ def aws2_add_region
     defined?(::Aws) ? { s3_region: 'us-east-1' } : {}
   end
 
+  context 'multithreaded initialization' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.